bindings/wasm/build/node.js

```diff
@@ -10,7 +10,7 @@ lintBigInt(entryFileNode);
 
 let changedFileNode = entryFileNode.replace(
     "let imports = {};",
-    "const fetch = require(\'node-fetch\')\r\nglobal.Headers = fetch.Headers\r\nglobal.Request = fetch.Request\r\nglobal.Response = fetch.Response\r\nglobal.fetch = fetch\r\n\r\nlet imports = {};"
+    "if (!global.is_fetch_polyfilled) {\r\nconst fetch = require(\'node-fetch\')\r\nglobal.Headers = fetch.Headers\r\nglobal.Request = fetch.Request\r\nglobal.Response = fetch.Response\r\nglobal.fetch = fetch\r\nglobal.is_fetch_polyfilled=true\r\n}\r\nlet imports = {};"
```

Comment:
I'm wondering if we could make that nicer to read with a multi-line string? Would get rid of the `\r\n`. Shouldn't that be possible with backticks in JS?

Reply:
Node supports them >= 4.0.0. So totally :)
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#browser_compatibility